packages/interface/src/store/deprecated.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import deprecated from '@wordpress/deprecated';
+
+export function normalizeComplementaryAreaScope( scope ) {
```

Comment:
This function ensures there's no breaking change for people calling these selectors/actions in their code base. I initially thought the interface package is entirely private but it turns out that folks can access its actions and selectors unfortunately.